printer_mods/Tircown/Display_mount_5inch/README.md

```diff
@@ -0,0 +1,25 @@
+# Voron 2.4 5" display tilting support
+**Many thanks to Jo_Thingiverse for having tested this mount.**
+
+### Bill Of Materials
+* Screen 5" 800*480 IPS TFT MIPI DSI Multi-Touch Capacitive Touch Panel LCD Module Display Monitor Screen for Raspberry Pi
+* optional SD extender or microSD extender
+* 2 M3x15
+* 6(+4) M3x8
+* 6 M3 threaded inserts
+* 2 M3 hammered nuts
+
+### Instruction
+1. Put one M3 threaded insert in each arm end.
+2. Put four M3 threaded insert in the back of the screen case.
+3. Install the two slides with M3x8.
+4. Use two pieces of filament as hinges.
+5. Adjust the tension of the screws that pass through the slides to keep the position.
+6. The optional installation of the SD extender requires four self tapping M3x8.
+
+**Since the screen is larger than the original piece you will have to replace your sides skirts. These are included too.**
+
```

Comment:
Is this cavity supposed to be there in `display5inch_case.stl` (and `display5inch_case_moretolerance.stl`)?  That wall covering it is really thin
<img width="326" alt="Screen Shot 2021-05-01 at 14 31 00" src="https://user-images.githubusercontent.com/69389593/116791691-1a657e80-aa8a-11eb-8cc8-6247d11d376e.png">

Reply:
Thank you for your comment. 
This cavity is there to guide the DSI thin cable.The two small tabs act as retainer, there is very little force applied to it. 
In lh 0.2mm, it slices well with 4 layers, imho sufficient for this use. 
Let me know if its ok for you, since it is not really required to hold the DSI cable, I can simply remove this whole feature.
![image](https://user-images.githubusercontent.com/74233386/116806248-fb1f2d80-ab2b-11eb-80aa-96b2c774c9ec.png)

Reply:
So there are two cavities, I want to make sure we're talking about the same one.  In the below picture, the one circled in blue is fine, and I'm asking about the thin wall circled in red.
<img width="756" alt="Screen Shot 2021-05-02 at 09 22 34" src="https://user-images.githubusercontent.com/69389593/116814720-4afbf580-ab28-11eb-885d-46309cb27cfa.png">

That second cavity itself is fine too, I'm concerned about a very thin wall blocking it.  It's hard to see in the pictures, but that wall is so thin that it disappears when sliced (even with "detect thin walls" turned on).

<img width="267" alt="Screen Shot 2021-05-02 at 09 28 11" src="https://user-images.githubusercontent.com/69389593/116814852-c2318980-ab28-11eb-8091-3936b529ce01.png">
<img width="236" alt="Screen Shot 2021-05-02 at 09 28 19" src="https://user-images.githubusercontent.com/69389593/116814857-c65da700-ab28-11eb-88b0-19d72c628a23.png">
If that second cavity is supposed to be there, then can you remove the thin wall in front of it?

Reply:
I got it. Sorry I didn't see this wall, it's a bug. I thought you talk about the little legs. I will correct it.